### DISCUSSION TITLE
IR receiver mode - "DataLSB"?

### DISCUSSION THREAD

**Discussion by ed2007** [arendst/Tasmota#18779] [Q&A] [answered]
Hi, IR receiver mode: does not show "DataLSB" in the console, can this be enabled somehow?
Thank u so much.
best regards

**Comment by s-hadinger** [comment]
Hi, you can't expect any decent answer while providing so little details. At least logs, version...

**Reply by ed2007** [reply]
12.5.0(tasmota) build:2023-04-17T08:03:54
Updated via otg today
This is not a malfunction, the authors decided that this is not the right data. I ask - is there a way to enable their display, if not - I'll have to go back to the old version.

**Reply by sfromis** [reply]
You still need to provide details, like logs, and also why you say "the authors decided".

When using IR receiver functionality, the payload I see does contain DataLSB with latest dev version 12.5.0.4
`{"IrReceived":{"Protocol":"RC5","Bits":12,"Data":"0x1DB","DataLSB":"0x80DB","Repeat":0}}`

**Comment by ed2007** [comment]
I do not quite understand what data should be shown? 
`20:46:58.295 RSL: RESULT = {"Time":"2023-06-02T20:46:58","IrReceived":{"Protocol":"NEC","Bits":32,"Data":"0x4DB2738C"}}
20:48:51.394 RSL: STATE = {"Time":"2023-06-02T20:48:51","Uptime":"0T00:05:08","UptimeSec":308,"Heap":24,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Wifi":{"AP":1,"SSId":"xx","BSSId":"xx:FE:ED:CD:AA:FC","Channel":11,"Mode":"11n","RSSI":86,"Signal":-57,"LinkCount":1,"Downtime":"0T00:00:03"}} `

**Reply by sfromis** [reply]
On what basis do you say "the authors decided"?

**Reply by s-hadinger** [reply]
Now I get it. In minimal IR mode, `DataLSB` is not present indeed, while it is in IR-full mode. Let me check if it's easy to add to IR-minimal

**Reply by s-hadinger** [reply, marked as answer]
Sorry, I took a look but IR-minimal is really old code. It would need a major rewrite to print and accept DataLSB. Please move to tasmota-ir which provides all the good features for IR.

**Reply by ed2007** [reply]
Thank you for your reply. The most amazing thing is that I had a full ir version after the update, the update by air (otg)  became minimal )
Best regards, Ed.
